contracts/payloads/IGP81/PayloadIGP81.sol

```diff
@@ -0,0 +1,503 @@
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(this) == PROPOSER_AVO_MULTISIG_3 ||
+                address(this) == PROPOSER_AVO_MULTISIG_4 ||
+                address(this) == PROPOSER_AVO_MULTISIG_5,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set dust limits for ezETH-ETH DEX and ezETH<>wstETH T1 & ezETH-ETH<>wstETH T2 vaults
+        action1();
+
+        // Action 2: Increase sUSDe-USDT<>USDT supply and borrow cap 
+        action2();
+
+        // Action 3: Set dust allowance for cbBTC-USDT DEX T4 vault
+        action3();
+
+        // Action 4: Reduce max supply shares and max borrow limit for USDe-USDT<>USDT
+        action4();
+
+        // Action 5: Increase LTV & LT for USDe-USDT<>USDT
+        action5();
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Set dust limits for ezETH-ETH DEX and ezETH<>wstETH T1 & ezETH-ETH<>wstETH T2 vaults
+    function action1() internal {
+        address ezETH_ETH_DEX = getDexAddress(21);
+        address ezETH__wstETH_VAULT = getVaultAddress(103);
+        address ezETH_ETH__wstETH_VAULT = getVaultAddress(104);
+
+        {
+            // ezETH-ETH DEX
+            {
+                // ezETH-ETH Dex
+                Dex memory DEX_ezETH_ETH = Dex({
+                    dex: ezETH_ETH_DEX,
+                    tokenA: ezETH_ADDRESS,
+                    tokenB: ETH_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000, // $10k
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_ezETH_ETH); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(ezETH_ETH_DEX, TEAM_MULTISIG, true);
+            }
+        }
+
+        {
+            // [TYPE 1] ezETH<>wstETH | normal collateral & normal debt
+            Vault memory VAULT_ezETH_wstETH = Vault({
```

Comment:
on such code I'd prefer defining the variable within the `{` scope block, so the code itself won't even allow to set a wrong value. E.g. currently if setting 
```
vault: ezETH_ETH__wstETH_VAULT,
```
here would work just fine. But when defining
 ```        
 address ezETH__wstETH_VAULT = getVaultAddress(103);
 ```
 
 here and 
  ```
          address ezETH_ETH__wstETH_VAULT = getVaultAddress(104);
  ```
  only in next scope, possibility for human error is eliminated

Reply:
noted!